miniupnpd/miniupnpd.c

```diff
@@ -2901,6 +2913,8 @@ main(int argc, char * * argv)
 	freeoptions();
 #endif
 
+	shutdown_redirect();
```

Comment:
nice one !

Reply:
thanks for spoting that !

Reply:
Didn't really matter while shutdown_redirect() was a stub. So it was a problem of my own creation :)